### PR TITLE
added functionality to change category

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/modalBottomSheet/ModalBottomSheet.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+export const ModalBottomSheet = ({
+    buttonTitle,
+    buttonStyles,
+    children,
+}: PropsWithChildren<Props>) => {
+    const bottomSheetRef = useRef<BottomSheetModal>(null);
+    const { isOpen: isVisibleDimmer, onOpen: onShowDimmer, onClose: onHideDimmer } = useToggle();
+
+    const snap = useMemo(() => snapPoints, []);
```
Comment (vell28) [inline src/components/modalBottomSheet/ModalBottomSheet.tsx]: plese remove `snap `

File: src/components/selectCategoryHome/SelectCategoryHome.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    const { dismiss } = useBottomSheetModal();
+
+    const handleSelectedCategory = (id: string) => {
+        setCategory(id);
+        dismiss();
+    };
+
+    useEffect(() => {
+        const fetchData = async () => {
```
Comment (vell28) [inline src/components/selectCategoryHome/SelectCategoryHome.tsx]: why we need this const here?

File: src/components/modalBottomSheet/ModalBottomSheet.tsx (hunk 1 above)
Comment (Ksenaaa) [reply src/components/modalBottomSheet/ModalBottomSheet.tsx]: fixed

File: src/components/selectCategoryHome/SelectCategoryHome.tsx (hunk 2 above)
Comment (Ksenaaa) [reply src/components/selectCategoryHome/SelectCategoryHome.tsx]: fixed

**Pull request merged**